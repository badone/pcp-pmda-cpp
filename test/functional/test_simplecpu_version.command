./pmdasimplecpu --version | grep -v '^PCP version'
