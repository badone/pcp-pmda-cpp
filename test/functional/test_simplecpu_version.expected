
simplecpu PMDA
PMDA interface version 6

